client/src/components/Products.jsx

```diff
@@ -45,7 +45,7 @@ const GET_CATEGORIES = gql`
 `;
 
 const Products = () => {
-  const [_, setCategoryInput] = useState('all');
+  const [, setCategoryInput] = useState('all');
```

Comment:
Wdyt about creating folder with all constants?

Reply:
But I can't do this. React Hooks "useState" & "useQuery" cannot be called at the top level. React Hooks must be called in a React function component or a custom React Hook function.